syntax = "proto3";

package user_routes;

service UserRoutes{
    rpc tryLogin(Credentials) returns (LoginConfirm) {}
}

message Credentials{
    string username = 1;
    string password = 2;
    bool newAccount = 3;
}

message LoginConfirm{
    int32 status = 1;
    string token = 2;
}